### PR TITLE
[Plugin] Fix exception when calling os.makedirs

### PR DESCRIPTION
**Pull request by jcastill**
When calling get_cmd_output_path() to create a directory via the plugin hpssm, sos threw the following exception:

Traceback (most recent call last):
  File "/root/sos/sos/report/__init__.py", line 1224, in setup
    plug.setup()
  File "/root/sos/sos/report/plugins/hpssm.py", line 67, in setup
    logpath = self.get_cmd_output_path()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/sos/sos/report/plugins/__init__.py", line 2168,
	in get_cmd_output_path
    os.makedirs(cmd_output_path)
  File "<frozen os>", line 225, in makedirs
FileExistsError: [Errno 17] File exists:
'/var/tmp/sos.1gdy83zb/sosreport-localhost-vbwfnpn/sos_commands/hpssm'

This was happening because the directory 'hpssm' was already created. With this change we avoid any race where we call os.makedirs() after we have already created the plugin directory.

Closes: RHBZ #2216608

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

**Review by pmoravec — APPROVED** [review]
ACK. snap test failing is unrelated (there is a PR for it).

**Comment by arif-ali** [comment]
> ACK. snap test failing is unrelated (there is a PR for it).

I think the job never went through, someone with admin needs to restart that job ;)

**Comment by TurboTurtle** [comment]
Seems Cirrus is having some issues, I'm restarting the jobs again. This looks perfectly fine, but I'm gonna play it safe and wait for the tests to pass before merge. Additionally, I think this is important for 4.5.5, which is supposed to close today. I mentioned this on IRC but I'll say it here too - I'm going to try and close out 4.5.5 later today but at the latest it will be tomorrow.

**Review by arif-ali — COMMENTED** [review]
ack, lgtm :)

**Pull request merged**
